REFERENCE.md

```
@@ -6768,3 +6768,284 @@ scalekit_client.actions.providers.delete_custom_provider(
 </dd>
 </dl>
 </details>
+
+## Events
+
+<details><summary><code>scalekit_client.events.<a href="https://github.com/scalekit-inc/scalekit-sdk-python/blob/main/scalekit/events.py">list_events</a>(event_types?, start_time?, end_time?, organization_id?, source?, auth_request_id?, interceptor_id?, interceptor_status?, interceptor_decision?, connection_id?, connected_account_id?, page_size?, page_token?) -> ListEventsResponse</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)